### PR TITLE
improvement(pwa): improve caching strategies for POST requests

### PR DESCRIPTION
**Pull request by Arinono**

### CODE REVIEW

File: src/views/Login.vue [hunk 1]
```diff
@@ -193,7 +193,7 @@ export default class Login extends Vue {
     this.error = false
     fetch('/', {
       method: 'POST',
-      headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
+      headers: { 'Content-Type': 'application/x-www-form-urlencoded', 'Cache-Control': 'no-cache' },
```
Comment (williammartin) [inline src/views/Login.vue]: Do we want to keep this?

File: public/service-worker.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ * See https://goo.gl/nhQhGp
+ *
+ * The rest of the code is auto-generated. Please don't update this file
+ * directly; instead, make changes to your Workbox build configuration
+ * and re-run your build process.
+ * See https://goo.gl/2aRDsh
+ */
+
+importScripts('https://storage.googleapis.com/workbox-cdn/releases/4.3.1/workbox-sw.js')
+importScripts(
```
Comment (williammartin) [inline public/service-worker.js]: Where do we get this file name from? How do we know it is going to be the same?

File: src/views/Login.vue (hunk 1 above)
Comment (Arinono) [reply src/views/Login.vue]: We can remove it 😉

File: public/service-worker.js (hunk 2 above)
Comment (Arinono) [reply public/service-worker.js]: 🤷‍♂

Comment (williammartin) [reply public/service-worker.js]: Look, I'll accept this if it works, but I'm _very_ suspicious of this. It doesn't seem right at all.

**Comment by JeanBarriere** [comment]
I'm also suspecting there is something wrong about how we manage these here. Will search more about this when I come back so I can actually give a proper review to this

**Pull request closed without merging**